data/actionsmap/yunohost.yml

```diff
@@ -291,6 +291,17 @@ domain:
                     extra:
                         pattern: *pattern_domain
 
+        ### dns_conf()
+        dns-conf:
```

Comment:
Actually we use a `domain_xxx` pattern, so why don't call this function `domain_dns` ? It'll be consistent with the API route too (`GET /domain/<domain>/dns`).
Sorry I didn't catch it earlier.